modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -453,40 +434,26 @@ input CallForProposalsPropertiesInput {
   semester: Semester
 
   """
-  Optional RA limit start RA. Required only if raLimitEnd is specified. Nullable
-  on edit if raLimitEnd is also set to 'null'.
+  Coordinate limits.  If not specified, they will default according to the
+  coordinates that are safely visible during the active period of the call.
   """
-  raLimitStart: RightAscensionInput
+  coordinateLimits: SiteCoordinateLimitsInput
 
   """
-  Optional RA limit end RA. Required only if raLimitStart is specified. Nullable
-  on edit if raLimitStart is also set to 'null'.
+  Active period start date (inclusive) for this call.  Required on create.  The
+  active period for this call will start on the night of the provided `Date`
+  value. Must be before the `activeEnd` date.  Not nullable.  Limited to dates
+  between 1900 and 2100 (exclusive).
```

Comment:
The date range seems kind of arbitrary, but is necessary in order to use our LST calculation which is limited to those years.

Reply:
It will be close for release time